tests/system/src/test/java/test/robot/javafx/embed/swing/LinuxScreencastHangCrashTest.java

```diff
@@ -56,7 +56,6 @@ public class LinuxScreencastHangCrashTest {
 
     @BeforeAll
     public static void init() throws Exception {
-        Assumptions.assumeTrue(!Util.isOnWayland()); // JDK-8335470
```

Comment:
You need to replace this with a version check for JDK >= 24 rather than just removing it or else this will fail (probably hang) when the tests are run on JDK 23. See [this JBS comment](https://bugs.openjdk.org/browse/JDK-8335470?focusedId=14722937&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14722937).

Reply:
Updated.